smac/utils/util_funcs.py

```diff
@@ -16,6 +16,11 @@ def get_types(config_space, instance_features=None):
             types[i] = n_cats
             bounds[i] = (int(n_cats), np.nan)
 
+        elif isinstance(param, (OrdinalHyperparameter)):
+            n_cats = len(param.sequence)
```

Comment:
@mlindauer getting the bounds should be the same, should it not?
If not, what would be the correct way of doing it?

I'll take another look into how pyrfrs data container actually processes this information and if there's a special way of dealing with ordinals.